dev-docs/source/SampleLogsDev.rst

```diff
@@ -0,0 +1,148 @@
+| Time                 | Boolean flag value  | Description        |
++======================+=====================+====================+
+| 2020-Oct-20 12:19:40 | 0                   | -                  |
++----------------------+---------------------+--------------------+
+| 2020-Oct-20 12:19:41 | 1                   | nth period started |
++----------------------+---------------------+--------------------+
+| 2020-Oct-20 12:19:48 | 0                   | nth period ended   |
++----------------------+---------------------+--------------------+
+
+running
+~~~~~~~~~~~
+
+This log contains a series of times and boolean describing when a data collection was occurring, for example:
+
++----------------------+---------------------+--------------------------+
+| Time                 | Boolean flag value  | Description              |
++======================+=====================+==========================+
+| 2020-Oct-20 12:19:40 | 0                   | -                        |
++----------------------+---------------------+--------------------------+
+| 2020-Oct-20 12:19:41 | 1                   | Data collection started  |
++----------------------+---------------------+--------------------------+
+| 2020-Oct-20 12:19:46 | 0                   | Data collection ended    |
++----------------------+---------------------+--------------------------+
+
+periods
+~~~~~~~~
+
+This log contains times and period numbers describing when each period started, e.g.
+
++----------------------+---------------------+--------------------------+
+| Time                 | Period number       | Description              |
++======================+=====================+==========================+
+| 2020-Oct-20 12:19:40 | 1                   | Period 1 started         |
++----------------------+---------------------+--------------------------+
+| 2020-Oct-20 12:19:49 | 2                   | Period 2 started         |
++----------------------+---------------------+--------------------------+
+| 2020-Oct-20 12:19:56 | 3                   | Period 3 started         |
++----------------------+---------------------+--------------------------+
+
+These three logs are constructed from the ICP_EVENT which tells you when various “events” of significance to the
+software happen, such as beginning or end of a run or a change of period number.
+
+
+Filtering
+~~~~~~~~~~
+
+For multiperiod workspaces, time-series data, such as Theta will be filtered by combining the `period n` and `running`
+logs, to create a filter describing when data collection for each period occurred. For example, a filter for the nth
+period would be described as follows, where we note that each period is described entirely be two boolean values, the
+first describing when the data collection started for that period and the second stating when data collection ended.
+
++----------------------+---------------------+-----------------------------------------+
```

Comment:
It this is an example of `Theta`, then Theta is a Float series, in the workspaces loaded by the example script they do not look like this.

Reply:
The table below is the structure of the filter object used to filter data such as Theta, I can reword it if you think it would be clearer

Reply:
On reflection the way you have put it already if fine